tools/wptserve/wptserve/server.py

```diff
@@ -342,6 +345,34 @@ def handle_connect(self, response):
             self.setup()
         return
 
+    def log_request(self, code="-", size="-"):
+        if isinstance(code, http.HTTPStatus):
+            code = code.value
+
+        self.logger.debug(
+            "{} - - [{}] {!r} {!s} {!s}".format(
+                self.address_string(),
+                self.log_date_time_string(),
```

Comment:
Shouldn't the logger add the timestamp? Not a blocker however.

Reply:
I was just copying what the superclass does, honestly.